### PR TITLE
Update to v2.0.0

### PR DESCRIPTION
**Pull request by JTD420**
- Added GUI and few other features and bug fixes
- Temporarily removed compatibility with MacOS and Linux while I figure out how to cross-compile with wails.

### CODE REVIEW

**Comment by JTD420** [comment]
hmm where'd I go wrong? haha

File: store/extensions/[AIO] Gamba Suite/extension.json [hunk 1]
```diff
@@ -46,40 +46,14 @@
       "-f",
       "{filename}"
     ],
```
Comment (WiredSpast) [inline store/extensions/[AIO] Gamba Suite/extension.json]:
```suggestion
    ]
```

**Comment by JTD420** [comment]
all good?

**Comment by JTD420** [comment]
@WiredSpast

**Pull request merged**